observer/observer.go

```diff
@@ -0,0 +1,113 @@
+package observer
+
+import (
+	"context"
+	"errors"
+
+	errorsmod "cosmossdk.io/errors"
+	abcitypes "github.com/cometbft/cometbft/abci/types"
+	"github.com/cometbft/cometbft/libs/log"
+	"github.com/cometbft/cometbft/libs/pubsub/query"
+	rpchttp "github.com/cometbft/cometbft/rpc/client/http"
+	coretypes "github.com/cometbft/cometbft/rpc/core/types"
+	comettypes "github.com/cometbft/cometbft/types"
+)
+
+const ModuleNameObserver = "observer"
+
+type Observer struct {
+	logger        log.Logger
+	tmRpc         *rpchttp.HTTP
+	stopChan      chan struct{}
+	eventsOutChan chan abcitypes.Event
+}
+
+// NewObserver returns new instance of `Observer` with RPC client created
+func NewObserver(logger log.Logger, rpcUrl string) (Observer, error) {
+	if len(rpcUrl) == 0 {
+		return Observer{}, errors.New("RPC URL can't be empty")
+	}
+
+	rpc, err := rpchttp.New(rpcUrl, "/websocket")
+	if err != nil {
+		return Observer{}, errorsmod.Wrapf(err, "Failed to create RPC client")
+	}
+
+	return Observer{
+		logger:        logger.With("module", ModuleNameObserver),
+		tmRpc:         rpc,
+		stopChan:      make(chan struct{}),
+		eventsOutChan: make(chan abcitypes.Event),
+	}, nil
+}
+
+// Starts RPC client, subscribes to events for provided query and starts listening to the events
```

Comment:
The comment doesn't specify the godoc. godoc requires that all method comments start with the name of the method. E.i., 
```suggestion
// Start starts the RPC client, subscribes to events for provided query and starts listening to the events
```
Please make sure to correct it everywhere